lib/logstash/inputs/rabbitmq.rb

```diff
@@ -131,17 +139,87 @@ def consume!
         end
       end
 
+      public
       def stop
         super
         shutdown_consumer
         close_connection
       end
 
+      public
```

Comment:
What is the preferred style? Put all private methods at the end of the class and precede that group of methods with a single `private`?